aspnetcore/fundamentals/index.md

```diff
@@ -4,17 +4,22 @@ author: rick-anderson
 description: Discover the foundational concepts for building ASP.NET Core apps.
 ms.author: riande
 ms.custom: mvc
-ms.date: 08/20/2018
+ms.date: 10/23/2018
 uid: fundamentals/index
 ---
 # ASP.NET Core fundamentals
 
-An ASP.NET Core app is a console app that creates a web server in its `Main` method:
+An ASP.NET Core app is a console app that creates a web server in its `Program.Main` method. The `Main` method is the app's *managed entry point*:
 
 ::: moniker range=">= aspnetcore-2.0"
 
 [!code-csharp[](index/snapshots/2.x/Program.cs)]
 
+The [.NET Core Host](/dotnet/core/tutorials/netcore-hosting), usually *dotnet.exe*:
+
+* Loads the [Core Common Language Runtime (CoreCLR)](https://github.com/dotnet/coreclr).
+* Searches the project's files for the managed entry point and begins code execution.
```

Comment:
The dotnet.exe does not *search* for managed entrypoint in the project files. The managed entrypoint is always in the binary that you are executing, not in some other random project file.

E.g. if you run `dotnet foo.dll`, the managed entrypoint will be in `foo.dll`, it cannot be anywhere else.